### PR TITLE
Release 3.1.0

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Review by blackbaud-johnly — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.1.0 (2019-05-28)
+
+- Added the ability to trigger responsive styles based on a parent component. [#9](https://github.com/blackbaud/skyux-link-records/pull/9)
+- Fixed the link records component to update messages. [#7](https://github.com/blackbaud/skyux-link-records/pull/7)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md]: The last one was a little hard to understand - I'm not sure what its trying to say. Should it say something like "Updated the UI text for the link records component" ?

Comment (blackbaud-johnly) [reply CHANGELOG.md]: That works for me. I'd changed "to have correct messages" to "to update messages," but sounds like that changes wasn't enough. I'll go ahead and make the change you suggest. We can tweak it further if anyone disagrees with that change.

Comment (blackbaud-johnly) [reply CHANGELOG.md]: Actually, looking back at the PR for this (https://github.com/blackbaud/skyux-link-records/pull/7), it looks like the text isn't really just "UI text." I'm going to go with "Updated UI text and messages for the link records component"

**Review by blackbaud-johnly — APPROVED** [review]
LGTM

**Pull request merged**